### ISSUE TITLE
Open in Browser: the url is wrong when working with non ascii words

### ISSUE THREAD

**Issue by aristotll** [ben-gibson/jetbrains-open-in-git-host#40]
The hosts is gitlab and I use OSX.
When I have non ascii words in my directory name, the url is wrong.

*需求* means *requirement* in Chinese.
When I open in git host with **0928.sql** in folder **需求0920-2017**.

```bash
tree -N .
.
└── 需求0920-2017
    └── 0928.sql

1 directory, 1 file
```

The generated url in browers is **需求0920-2017/0928.sql%23L15**
And the right version should be **需求0920-2017/0928.sql#L15**.

It seems there is a wrong url encoding converting `#` to `%23`.

---

Also the result of *Copy to Clipboard* is right: **需求0920-2017/0928.sql#L15**

Thanks.

**Comment by ben-gibson** [comment]
Thanks! This should be fixed in `v2.0.1` which has just been released. Let me know if that fixes the issue for you.

**Comment by aristotll** [comment]
@ben-gibson  The bug is fixed, thanks.
